src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -300,12 +300,18 @@ def _change_org_member_role(member: OrganizationMember, role: str) -> None:
             # null. We do this because such a team role would be effectively
             # invisible in the UI, and would be surprising if it were left behind
             # after the user's org role is lowered again.
-            OrganizationMemberTeam.objects.filter(
+            omt_update_count = OrganizationMemberTeam.objects.filter(
                 organizationmember=member, role__in=lesser_team_roles
             ).update(role=None)
 
             member.update(role=role)
 
+        if omt_update_count > 0:
+            metrics.incr(
+                "team_roles:update_to_detault",
```

Comment:
I don't think datadog supports `:` as a character here, looks like it'll get converted to an underscore https://docs.datadoghq.com/developers/guide/what-best-practices-are-recommended-for-naming-metrics-and-tags/

Reply:
Oops, those colons were meant to be periods. Thanks for the catch.